app/poros/github_users.rb

```diff
@@ -0,0 +1,27 @@
+class GithubUsers
+
+  def repos
+    body = GithubService.new.repos
+    body.map do |repo_data|
+      Repo.new(repo_data)
+    end
+  end
+
+  def repos_limit(number)
+    repos.first(number)
```

Comment:
thanks for this suggestion! It is much cleaner than what we had put there initially.